wp-content/themes/core/components/image/Controller.php

```diff
@@ -117,7 +117,7 @@ public function __construct( array $args = [] ) {
 		$this->use_srcset = $args['use_srcset'] ?? true;
 		$this->wrapper_attrs = (array) ( $args['wrapper_attrs'] ?? [] );
 		// pass classes for figure wrapper. If as_bg is set true gets auto class of "lazyload"
-		$this->wrapper_classes = (array) ( $args['classes'] ?? [ 'c-image' ] );
+		$this->wrapper_classes = array_merge( [ 'c-image' ], (array) ( $args['classes'] ?? [] ) );
```

Comment:
Is there styles associated with `c-image`? This would be once case where I don't want to have to override any default styles.

Reply:
Not totally sure yet. but given it has so many options and things going on, at least to start, I don't think it hurts to have a commonly available css hook available to do all the things with. We can def revise if we find in project use it goes the way of the painful button 🙃